lib/problem_check/twitter_login.rb

```diff
@@ -2,19 +2,22 @@
 
 class ProblemCheck::TwitterLogin < ProblemCheck
   self.priority = "high"
-
-  # TODO: Implement.
   self.perform_every = 24.hours
+  self.max_blips = 3
 
-  def call
+  def call(tracker)
     return no_problem if !authenticator.enabled?
     return no_problem if authenticator.healthy?
 
+    authenticator.disable if disable_failing_social_logins? && self.class.max_blips >= tracker.blips
```

Comment:
Maybe something like `tracker.blip_threshold_reached?(self.class)`? Not super necessary just might read nicer

Reply:
I think the right API might be in `ProblemCheck`, but deferring until I have more 🍖.